server/src/main/java/com/sauron/service/TestEntitySerivce.java

```diff
@@ -0,0 +1,8 @@
+package com.sauron.service;
+
+import com.sauron.model.TestEntityPojo;
+
+public interface TestEntitySerivce {
```

Comment:
Serivce -> Service

Besides, personally I'm not a big fan of creating interfaces when only one implementation exists (unless there are some reasons behind that)

Reply:
Name changed.